packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -1207,7 +1207,7 @@ class _LineItemState extends State<LineItem>
               // to allow us to render this as a proper overlay as similar
               // functionality exists to render the selection handles properly.
               Opacity(
```

Comment:
what is the point of using a completely transparent opacity widget now?

Reply:
is the comment above this widget still relevant?

Reply:
Hmm the comment (and this code) is 2 years old: https://github.com/flutter/devtools/pull/2665. I can investigate the feasibility if you think that's better!

I think I accidentally made it semi-transparent when debugging https://github.com/flutter/devtools/pull/4651, and forgot to switch it back